src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/CwdsDAO.scala

```diff
@@ -1,9 +1,15 @@
 package org.broadinstitute.dsde.firecloud.dataaccess
 
-import org.broadinstitute.dsde.firecloud.model.{AsyncImportRequest, AsyncImportResponse, ImportServiceListResponse, UserInfo}
+import org.broadinstitute.dsde.firecloud.model.{AsyncImportRequest, ImportServiceListResponse, UserInfo}
 import org.databiosphere.workspacedata.client.ApiException
 import org.databiosphere.workspacedata.model.GenericJob
 
+object LegacyFileTypes {
```

Comment:
Pulled over from `ImportServiceDAO` which went away.

Reply:
👍 for renaming these